queries/python/fold.scm

```diff
@@ -0,0 +1,24 @@
+(function_definition (block) @fold)
+(class_definition (block) @fold)
+
+(while_statement (block) @fold)
+(for_statement (block) @fold)
+(if_statement (block) @fold)
+(with_statement (block) @fold)
+(try_statement (block) @fold)
+
+[
+  (import_from_statement)
+  (parameters)
+
+  (parenthesized_expression)
+  (generator_expression)
+  (list_comprehension)
+  (set_comprehension)
+  (dictionary_comprehension)
+
+  (tuple)
+  (list)
+  (set)
+  (dictionary)
```

Comment:
I don't like how these fold, but they don't have a node like `block`. Maybe always leaving the first line visible? I suspect many other grammars don't have an explicit node like `block`. Or maybe there is a way to capture everything inside the node?